webapp/src/modules/parcels/sagas.js

```diff
@@ -85,3 +89,27 @@ function* handleEditParcelsRequest(action) {
     yield put(editParcelFailure(parcel, error.message))
   }
 }
+
+function* handleManageParcelsRequest(action) {
+  try {
+    const { parcel, address, revoked } = action
+    const { x, y } = parcel
```

Comment:
_nitpick_: we can move this line above the `try` to avoid repeating it on the catch